website/docs/docs/dbt-versions/release-notes/04-Sept-2023/ci-updates-phase2-rn.md

```diff
@@ -29,4 +29,8 @@ Below is a comparison table that describes how deploy jobs and CI jobs behave di
 
 ## What you need to update
 
-If you previously set up a job using the [Create Job](/dbt-cloud/api-v2#/operations/Create%20Job) API endpoint before September 11, 2023, you must re-create the job as described in [Trigger a CI job with the API](/docs/deploy/ci-jobs#trigger-a-ci-job-with-the-api). This is because you must set the `job_type` to be `ci`. 
\ No newline at end of file
+- If you want to set up a CI environment for your jobs, dbt Labs recommends that you create your CI job in a dedicated [deployment environment](/docs/deploy/deploy-environments#create-a-deployment-environment) that's connected to a staging database. To learn more about these environment best practices, refer to the guide [Get started with continuous integration tests](/guides/orchestration/set-up-ci/overview). 
+
+- If you had set up a CI job before September 11, 2023 using the [Create Job](/dbt-cloud/api-v2#/operations/Create%20Job) API endpoint but didn't set `"triggers":triggers.git_provider_webhook`, you must re-create the job as described in [Trigger a CI job with the API](/docs/deploy/ci-jobs#trigger-a-ci-job-with-the-api). This is because the job was misclassified as a deploy job with this update.
+
+- If you previously set up a job using dbt Cloud before September 11, 2023, you must re-create the job as described in [Set up CI jobs](/docs/deploy/ci-jobs#set-up-ci-jobs) and make sure the **Triggered by pull requests** option is enabled. By default, it is.
```

Comment:
one more clarification here: if the user is trying to create a CI job in the new UI, they don't necessarily need PR triggers. they can just select `Continuous Integration Job` & it will be classified as CI (even if the job isn't triggered by pull requests)
![Screenshot 2023-09-20 at 13 42 39](https://github.com/dbt-labs/docs.getdbt.com/assets/56835998/6a5e8b2d-146f-473d-9922-efb244165b5a)

Reply:
ah ok, i can remove that clause then!